File: 백준/Silver/11399. ATM/ATM.cc
#include <iostream>
#include <algorithm>

using namespace std;

int main(void) {
    int n;
    cin >> n;
    
    int *time = new int[n];
    for (int i = 0; i < n; i++) {
        cin >> time[i];
    }
    sort(time, time + n);
    
    int result = 0;
    for (int i = 0; i < n; i++) {
        for (int j = 0; j <= i; j++) {
            result += time[j];
        }
    }
    cout << result << '\n';
    return 0;
}

File: 백준/Silver/11399. ATM/ATM.java
import java.util.Scanner;
import java.util.Arrays;

public class Main {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        
        int n = scanner.nextInt();
        
        int[] arr = new int[n];
        for (int i = 0; i < n; i++) {
            arr[i] = scanner.nextInt();
        }
        
        Arrays.sort(arr);
        
        int result = 0;
        for (int i = 0; i < n; i++) {
            int temp = 0;
            for (int j = 0; j <= i; j++) {
                temp += arr[j];
            }
            result += temp;
        }
        
        System.out.println(result);
    }
}